### PR TITLE
Update the supported installation way in README

### PR DESCRIPTION
**Pull request by tnozicka**
Closes https://github.com/redhat-developer/opencompose/issues/37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +21,18 @@ If you prefer to do it straight from CLI here is a one-liner for you:
 ```bash
 curl -L https://github.com/redhat-developer/opencompose/releases/download/v0.1.0-alpha.0/opencompose-v0.1.0-alpha.0-d0edfd9-linux-64bit.tar.xz | tar -xJf - -C ${HOME}/bin ./opencompose
 ```
+#### From source
+You can easily build OpenCompose tool from source. All you need is [Go installed](https://golang.org/dl/) on your system. Than simply run:
+```sh
+go get -u github.com/redhat-developer/opencompose
+```
+
+Although the simple `go get` installation works for just trying it out, if you want to submit bug reports you have to build it properly:
```
Comment (cdrage) [inline README.md]: Bit harsh saying that it's "built properly" this way.

IMO, there should be two sections:

```
### From source
go get -u foobar

### Manual installation
curl commands, blah blah
```

Saying that `go get` isn't suitable and that you *HAVE* to do make install doesn't seem fitting.

File: README.md [hunk 2]
```diff
@@ -27,6 +21,18 @@ If you prefer to do it straight from CLI here is a one-liner for you:
...
+```bash
+go get -u -d github.com/redhat-developer/opencompose
+make install -C $GOPATH/src/github.com/redhat-developer/opencompose/
+```
+This will make sure proper output of `opencompose version` and other features dependent on using linker flags.
```
Comment (cdrage) [inline README.md]: linker flags? May be too much "fluff" / advanced stuff here. What about saying that it's required to determine the git commit version?

**Review by cdrage — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tnozicka) [reply README.md]: I am ok with rephrasing if *properly* seem a bit too harsh, but the truth is that although `go get` works, it is not the supported way of building OpenCompose and you need to use `make install` (or `make build`) to have one that is supported. And of course all the binaries we build and publish on are release page are supported. (Supported means that we accept bug reports for it.)

File: README.md (hunk 2 above)
Comment (tnozicka) [reply README.md]: I wanted to avoid changing README if we decide to do any other advanced stuff in `make build` or `make install` but I'll rephrase it to be less advanced.

File: README.md [hunk 3]
```diff
@@ -27,7 +21,19 @@ If you prefer to do it straight from CLI here is a one-liner for you:
...
+```sh
+go get -u github.com/redhat-developer/opencompose
+```
 
+Although the simple `go get` installation works for just trying it out, if you want to submit bug reports you have to build it the supported way:
+```bash
+go get -u -d github.com/redhat-developer/opencompose
+make install -C $GOPATH/src/github.com/redhat-developer/opencompose/
+```
+This will make sure proper output of `opencompose version` which is dependant on advanced build.
```
Comment (kadel) [inline README.md]: "... is dependant on advanced build." - hmm, I don't know what that sentence means :-(

**Comment by tnozicka** [comment]
ping @kadel

File: README.md [hunk 4]
```diff
@@ -29,6 +23,18 @@ If you prefer to do it straight from CLI here's a one-liner for you:
...
+```sh
+go get -u github.com/redhat-developer/opencompose
+```
+
+Although the simple `go get` installation works for just trying it out, if you want to submit bug reports you have to build it the supported way:
+```bash
+go get -u -d github.com/redhat-developer/opencompose
+make install -C $GOPATH/src/github.com/redhat-developer/opencompose/
+```
+This will make sure proper output of `opencompose version`.
```
Comment (cdrage) [inline README.md]: `This will ensure proper output of`

File: README.md [hunk 5]
```diff
@@ -29,6 +23,18 @@ If you prefer to do it straight from CLI here's a one-liner for you:
 ```bash
 curl -L https://github.com/redhat-developer/opencompose/releases/download/v0.1.0-alpha.0/opencompose-v0.1.0-alpha.0-d0edfd9-linux-64bit.tar.xz | tar -xJf - -C ${HOME}/bin ./opencompose
 ```
+#### From source
+You can easily build OpenCompose tool from source. All you need is [Go](https://golang.org/dl/):
+```sh
+go get -u github.com/redhat-developer/opencompose
+```
+
```
Comment (cdrage) [inline README.md:31]: :+1:

File: README.md [hunk 6]
```diff
@@ -29,6 +23,18 @@ If you prefer to do it straight from CLI here's a one-liner for you:
...
+Although the simple `go get` installation works for just trying it out, if you want to submit bug reports you have to build it the supported way:
```
Comment (cdrage) [inline README.md]: `Although `go get` instalation works for just trying it out` may sound a bit better

**Review by cdrage — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (tnozicka) [reply README.md]: ok, dropping `the simple`

File: README.md (hunk 4 above)
Comment (tnozicka) [reply README.md]: sounds better ;)

**Comment by tnozicka** [comment]
bump @cdrage @kadel

**Comment by cdrage** [comment]
LGTM

**Review by cdrage — APPROVED** [review]

**Review by kadel — APPROVED** [review]
